app/src/main/java/edu/grinnell/appdev/grinnelldirectory/Activity/LoginActivity.java

```diff
@@ -35,9 +43,60 @@ protected void onCreate(Bundle savedInstanceState) {
      *
      * @param view LoginActivity's view
      */
-    @OnClick(R.id.signin)
-    public void signIn(View view) {
-        String usernameTxt = mUsernameEditText.getText().toString();
-        String passwordTxt = mPasswordEditText.getText().toString();
+    @OnClick(R.id.login)
+    void signIn(View view) {
+        String username = mUsernameEditText.getText().toString();
+        String password = mPasswordEditText.getText().toString();
+        User user = new User(username, password);
+        DBAPICaller dbapiCaller = new DBAPICaller(user, this);
+        dbapiCaller.authenticateUser();
+    }
+
+    @Override public void onSearchSuccess(List<Person> people) {}
+
+    /**
+     * Save credentials and move to MainActivity if login succeeded
+     *
+     * @param success whether the login was successful
+     * @param person model for the logged in user
+     */
+    @Override public void authenticateUserCallSuccess(boolean success, Person person) {
+        String username = mUsernameEditText.getText().toString();
+        String password = mPasswordEditText.getText().toString();
+        if (success) {
+            User.saveCredentials(getApplicationContext(), username, password);
```

Comment:
Use static imports to shorten the code here.

Also, unless the application's context is entirely necessary, just pass in `this` since an `Activity` is a `Context` (see other comments for detailed explanation)